<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #9a9a9a;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>15</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #9a9a9a;
}

QLabel {
    color: #2980b9;
    font-size: 16px;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>CODIGO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCodigo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #f2f2f2;
}

QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #5DADE2, stop: 1 #2980b9);
    color: #ffffff;
    border: 1px solid #2980b9;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #4DA8DB, stop: 1 #2471a3);
    border: 1px solid #2471a3;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #3C89B4, stop: 1 #1f618d);
    border: 1px solid #1f618d;
}

QLineEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QLineEdit:hover {
    border: 1px solid #3498db;
}

QLineEdit:focus {
    border: 1px solid #3498db;
    outline: none;
}

QWidget {
    background-color: #f2f2f2;
}

QLabel {
    color: #2980b9;
    font-size: 16px;
    font-weight: bold;
}

QLineEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
    color: #2980b9;
}

QLineEdit::placeholder {
    color: #2980b9;
    font-weight: bold;
}

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>15</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #9a9a9a;
}

QLabel {
    color: #2980b9;
    font-size: 16px;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>NOMBRE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #f2f2f2;
}

QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #5DADE2, stop: 1 #2980b9);
    color: #ffffff;
    border: 1px solid #2980b9;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #4DA8DB, stop: 1 #2471a3);
    border: 1px solid #2471a3;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #3C89B4, stop: 1 #1f618d);
    border: 1px solid #1f618d;
}

QLineEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QLineEdit:hover {
    border: 1px solid #3498db;
}

QLineEdit:focus {
    border: 1px solid #3498db;
    outline: none;
}

QWidget {
    background-color: #f2f2f2;
}

QLabel {
    color: #2980b9;
    font-size: 16px;
    font-weight: bold;
}

QLineEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
    color: #2980b9;
}

QLineEdit::placeholder {
    color: #2980b9;
    font-weight: bold;
}

</string>
    </property>
   </widget>
   <widget class="QTableView" name="dataView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>71</y>
      <width>371</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView {
    background-color: #9a9a9a;
    border: 1px solid #cccccc;
    selection-background-color: #3498db;
    selection-color: #ffffff;
}

QTableView QHeaderView {
    background-color: #d3d3d3;
    color: #333333;
    border: none;
}

QTableView QHeaderView::section {
    background-color: #bbbbbb;
    color: #333333;
    padding: 6px;
    border: 1px solid #cccccc;
}

QTableView QHeaderView::section:checked {
    background-color: #3498db;
    color: #ffffff;
}

QTableView::item {
    padding: 6px;
}

QTableView::item:selected {
    background-color: #3498db;
    color: #ffffff;
}

QTableView::item:focus {
    background-color: #3498db;
    color: #ffffff;
    outline: none;
}

QTableView::item:hover {
    background-color: #d3d3d3;
    color: #333333;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGuardar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #5DADE2, stop: 1 #2980b9);
    color: #ffffff;
    border: 1px solid #2980b9;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #4DA8DB, stop: 1 #2471a3);
    border: 1px solid #2471a3;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #3C89B4, stop: 1 #1f618d);
    border: 1px solid #1f618d;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../png/Guardar.png</normaloff>../png/Guardar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #5DADE2, stop: 1 #2980b9);
    color: #ffffff;
    border: 1px solid #2980b9;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #4DA8DB, stop: 1 #2471a3);
    border: 1px solid #2471a3;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #3C89B4, stop: 1 #1f618d);
    border: 1px solid #1f618d;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../png/Exit.png</normaloff>../png/Exit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Esc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #5DADE2, stop: 1 #2980b9);
    color: #ffffff;
    border: 1px solid #2980b9;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #4DA8DB, stop: 1 #2471a3);
    border: 1px solid #2471a3;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #3C89B4, stop: 1 #1f618d);
    border: 1px solid #1f618d;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../png/limpiar.png</normaloff>../png/limpiar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBorrar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #5DADE2, stop: 1 #2980b9);
    color: #ffffff;
    border: 1px solid #2980b9;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #4DA8DB, stop: 1 #2471a3);
    border: 1px solid #2471a3;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #3C89B4, stop: 1 #1f618d);
    border: 1px solid #1f618d;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../png/borrar.png</normaloff>../png/borrar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
